apps/scan/backend/src/scanners/custom/state_machine.ts

```diff
@@ -370,6 +371,11 @@ function storeInterpretedSheet(
   return addedSheetId;
 }
 
+/**
+ * A mutex to ensure that continuous export operations happen sequentially and do not interleave
+ */
+const continuousExportMutex = new Mutex(0);
```

Comment:
This feels like it should be a class/object property rather than a module-level global variable.

Reply:
Good call! Made it a local variable within the `buildMachine` function: https://github.com/votingworks/vxsuite/pull/4365/commits/ce81444b6d527d81a2286217017d8a8b9a243d64

Reply:
Good point - this is making me realize that the final export operation handled by the polls-close operation should also be mutex-gated. Let me move the mutex up to the level of the `Store` or the `Workspace`

Reply:
https://github.com/votingworks/vxsuite/pull/4365/commits/8fac56efb2b8f0ed34096da940f54665b61c072e